action-server/tests/actions/test_checkin_return_form.py

```diff
@@ -202,22 +206,101 @@ def _test_moderate_symptoms(self, fever: bool):
         self.assert_events(
             [
                 SlotSet(MODERATE_SYMPTOMS_SLOT, True),
+                SlotSet(REQUESTED_SLOT, MODERATE_SYMPTOMS_WORSENED_SLOT),
+            ]
+        )
+
+        self.assert_templates(["utter_ask_checkin_return__moderate_symptoms_worsened"])
+
+    def test_fever_moderate_symptoms_worsened(self):
+        self._test_moderate_symptoms_worsened(True)
+
+    def test_no_fever_moderate_symptoms_worsened(self):
+        self._test_moderate_symptoms_worsened(False)
```

Comment:
Use fever=True and fever=False as arguments, just using True or False is confusing.